src/main/java/Dul.java

```diff
@@ -0,0 +1,181 @@
+import java.util.Scanner;
+
+public class Dul {
+    public static final int maximumTASKS = 100;
+    public static Task[] tasks = new Task[maximumTASKS];
+    public static int taskCount = 0;
+
+    public static void main(String[] args) {
+        String logo = " ____        _\n"
+                + "|  _ \\ _   _| |\n"
+                + "| | | | | | | |\n"
+                + "| |_| | |_| | |\n"
+                + "|____/ \\__,_|_|\n";
+        System.out.println("Hello I'm\n" + logo);
+        System.out.println("What can I do for you?");
+        String guyInput = "";
+        Scanner in = new Scanner(System.in);
+
+        while (!guyInput.equals("bye")) {
+            guyInput = in.nextLine();
+            listInput(guyInput);
+        }
+
+        System.out.println("Bye. Hope to see you again soon!");
+        in.close();
+    }
+
+    public static void listInput(String input) {
+        String[] command = input.split(" ", 2);
+        switch (command[0]) {
+            case "list":
+                listTasks();
+                break;
+            case "mark":
+                markTaskDone(Integer.parseInt(command[1]) - 1);
+                break;
+            case "unmark":
+                markTaskNotDone(Integer.parseInt(command[1]) - 1);
+                break;
+            case "todo":
+                addTodoTask(command[1]);
+                break;
+            case "deadline":
+                addDeadlineTask(command[1]);
+                break;
+            case "event":
+                addEventTask(command[1]);
+                break;
+            case "bye":
+                break;
+            default:
+                addTask(command[0]);
+                break;
+        }
+    }
```

Comment:
Great use of cases here it is very easy to read and understand the different cases at one glance